packages/experimental/Expander/src/Expander.types.ts

```diff
@@ -0,0 +1,161 @@
+import { ColorValue } from 'react-native';
+
+export const expanderName = 'Expander';
+
+/*
+  * General notes:
+  * The Expander header will be consumed as the first child element of the Expander
+  * The Expander content will be consumed as the second child element of the Expander
+  * To include more complex layouts for the header or content, wrap multiple header/content elements inside of a <View>
+  */
+
+export interface ExpanderProps {
+  /*
+   * Direction to expand the Expander.
+   */
+  expandDirection?: ExpandDirection;
+  /*
+   * Determines if the expander is currently expanded
+   */
+  expanded?: boolean;
+  /*
+   * Is Expander control enabled for the user
+   */
+  enabled?: boolean;
+  /*
+   * Height of the Expander
+   */
+  height?: number;
+  /*
+  * Height for the Expander when expanded
+  */
+  expandedHeight: number;
+  /*
+    * Height for the Expander when collapsed
+    */
+  collapsedHeight: number;
+  /*
+   * A callback to call on Expander collapsed event
+   */
+  onCollapsing?: () => void;
+  /*
+   * A callback to call on Expander expanding event
+   */
+  onExpanding?: () => void;
+}
+
+export interface ExpanderTokens {
+  /*
+   * Width of the expander
+   */
+  width?: number;
+  /*
+   * Horizontal alignment of the expander's content
+   */
+  contentHorizontalAlignment?: HorizontalAlignment;
+  /*
+   * Vertical alignment of the expander's content
+   */
+  contentVerticalAlignment?: VerticalAlignment;
+  /*
+   * Header background color
+   */
+  headerBackground?: ColorValue;
+  /*
+   * Header foreground color at rest
+   */
+  headerForeground?: ColorValue;
+  /*
+   * Header foreground color on pointer over
+   */
+  headerForegroundPointerOver?: ColorValue;
+  /*
+   * Header foreground color when pressed
+   */
+  headerForegroundPressed?: ColorValue;
+  /*
+   * Header border color at rest
+   */
+  headerBorderBrush?: ColorValue;
+  /*
+   * Header border color on pointer over
+   */
+  headerBorderPointerOverBrush?: ColorValue;
+  /*
+   * Header border color when pressed
+   */
+  headerBorderPressedBrush?: ColorValue;
+  /*
+   * Header foreground color when disabled
+   */
+  headerDisabledForeground?: ColorValue;
+  /*
+   * Header border color when disabled
+   */
+  headerDisabledBorderBrush?: ColorValue;
+  /*
+   * Header border thickness
+   */
+  headerBorderThickness?: Number;
+  /*
+   * Content background color
+   */
+  contentBackground?: ColorValue;
+  /*
+   * Content border color
+   */
+  contentBorderBrush?: ColorValue;
+  /*
+   * Chevron background color at rest
+   */
+  chevronBackground?: ColorValue;
+  /*
+   * Chevron foreground color at rest
+   */
+  chevronForeground?: ColorValue;
+  /*
+   * Chevron background color on pointer over
```

Comment:
the chevron gets different background and foreground colors from the rest of the control's `background`, e.g. we can't reuse BackgroundTokens ?

the variations should be handled in styling and only need chevronBackground chevronForeground

Reply:
I pulled these tokens from values that can be put in the [ResourceDictionary](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.expander?view=winui-2.6#remarks), so they can all be different colors.